### DISCUSSION TITLE
How can I set the status line to show the macro or operators?

### DISCUSSION THREAD

**Discussion by JuanZoran** [folke/noice.nvim#246] [Q&A] [answered]
I just install this plugin, and I set the noice with vim-notify and nui-vim
but I can solve this problem: 
- the status line show nothing when I press "q" + character for using macro
- it seems that everykey would be silent for that 

is this a bug? or just I configured this plugin with wrong
I will appreciate it if anyone would help me....

**Comment by folke** [comment, marked as answer]
https://github.com/folke/noice.nvim/wiki/A-Guide-to-Messages#showmode

**Comment by JuanZoran** [comment]
Thank you for reply my question
By the way, this is really a fancy plugin! :)

**Comment by ghost** [comment]
> https://github.com/folke/noice.nvim/wiki/A-Guide-to-Messages#showmode


`require("noice").api.statusline.mode.get` is deprecated now. Use `require("noice").api.status.mode.get` instead.


![deprecated](https://user-images.githubusercontent.com/82650811/223307968-2947c705-dfc5-4720-8ca6-4fc9d02ed377.png)
